docs/getting-started-overview.md

```diff
@@ -4,38 +4,43 @@ title: Getting Started with single-spa
+single-spa apps consist of the following:
 
-1. Many [applications](building-applications.md), each of which is sort of like an entire SPA itself. Applications respond to url routing events and must know how to bootstrap, mount, and unmount themselves from the DOM. The main difference between a SPA and an application is that applications must coexist together and do not each have their own html page.
-  For example, your React or Angular applications are applications which are either active or dormant. When active, they listen to url routing events and put content on the DOM. When dormant, they do not listen to url routing events and are totally removed from the DOM.
-2. A [single-spa-config](single-spa-config.md). The single-spa-config is the html page, plus the javascript that registers applications with single-spa. Each application is registered with three things:
-    1. A name
-    2. A function to load the application's code
-    3. A function that determines when the application is active/dormant.
+1. [Applications](building-applications.md), each of which is an entire SPA itself (sort of). Each application can respond to url routing events and must know how to bootstrap, mount, and unmount themselves from the DOM. The main difference between a traditional SPA and single-spa applications is that they must be able to coexist with other applications, and they do not each have their own html page.
+
+    For example, your React or Angular SPAs are applications. When active, they listen to url routing events and put content on the DOM. When inactive, they do not listen to url routing events and are totally removed from the DOM.
+2. A [single-spa-config](single-spa-config.md), which is the html page _and_ the JavaScript that registers applications with single-spa. Each application is registered with three things:
+    - A name
+    - A function to load the application's code
+    - A function that determines when the application is active/inactive
 
 ## How hard will it be to use single-spa?
 
-single-spa works with es5, es6+, typescript, webpack, systemjs, gulp, grunt, bower, ember-cli, or really any build system you can think of. You can npm install it, jspm install it, or even just use a `<script>` tag if you prefer. If you're not starting your application from scratch, you'll have to [migrate your SPA](migrating-existing-spas.md) to become a single-spa application.
+single-spa works with ES5, ES6+, TypeScript, Webpack, SystemJS, Gulp, Grunt, Bower, ember-cli, or really any build system available. You can npm install it, jspm install it, or even just use a `<script>` tag if you prefer.
+
+Our objective is to make using single-spa as easy as possible. But we should also point out that this is an advanced architecture that is different from how front-end developers are typically done. This may mean thinking about front-end applications differently and making corresponding changes. 
```

Comment:
> different from how front-end developers are typically done.

Maybe it should be "how front-end architecture is typically done." ?

Reply:
Fixed.